tests/manage/ocs_288.py

```diff
@@ -0,0 +1,292 @@
+    assert delete_cephblockpool()
+    log.info("Deleting RBD Secret")
+    RBD_SECRET_OBJ.delete()
+    log.info("Deleting CEPHFS Secret")
+    CEPHFS_SECRET_OBJ.delete()
+    log.info("Deleting CEPH FILESYSTEM")
+    CEPHFS_OBJ.delete()
+    log.info("Deleting Storageclass")
+    assert delete_storageclass()
+
+
+def delete_pvc():
+    """
+    Function to delete pvc
+    :param pvc_list:
+    :return:
+    """
+    pvc_list = get_pvc()
+    for item in pvc_list:
+        log.info(f"Deleting pvc {item}")
+        assert PVC.delete(resource_name=item)
+    return True
+
+
+def delete_storageclass():
+    """
+    Function for deleting Storageclass and CephBlockPool
+    :param storageclass_list:
+    :return:
+    """
+    storageclass_list = get_storageclass()
+    for item in storageclass_list:
+        log.info(f"Deleting StorageClass with name {item}")
+        assert SC.delete(resource_name=item)
+    return True
+
+
+def delete_cephblockpool():
+    """
+    Function for deleting CephBlockPool
+    :return:
+    """
+    pool_list = get_cephblockpool()
+    for item in pool_list:
+        log.info(f"Deleting CephBlockPool with name {item}")
+        assert POOL.delete(resource_name=item)
+    return True
+
+
+def create_cephfilesystem():
+    """
+    Function for deploying CephFileSystem (MDS)
+    :param kwargs:
+    :return:
+    """
+    fs_data = defaults.CEPHFILESYSTEM_DICT.copy()
+    fs_data['metadata']['name'] = helpers.create_unique_resource_name(
+        'test', 'cephfs'
+    )
+    fs_data['metadata']['namespace'] = ENV_DATA['cluster_namespace']
+    global CEPHFS_OBJ
+    CEPHFS_OBJ = OCS(**fs_data)
+    CEPHFS_OBJ.create()
+    assert POD.wait_for_resource(
+        condition='Running', selector='app=rook-ceph-mds'
+    )
+    assert validate_cephfilesystem(fs_name=fs_data['metadata']['name'])
+    return True
+
+
+def validate_cephfilesystem(fs_name):
+    """
+    Function for validating CephFileSystem Creation
+    :param fs_name:
+    :return:
+    """
+    tools_pod = pod.get_ceph_tools_pod()
+    ceph_validate = False
+    k8s_validate = False
+    cmd = "ceph fs ls"
+    log.info(fs_name)
+    out = tools_pod.exec_ceph_cmd(ceph_cmd=cmd)
+    if out:
+        out = out[0]['name']
+        log.info(out)
+        if out == fs_name:
+            log.info("FileSystem got created from Ceph Side")
+            ceph_validate = True
+        else:
+            log.error("FileSystem was not present at Ceph Side")
+            return False
+    result = CFS.get(resource_name=fs_name)
+    if result['metadata']['name']:
+        log.info(f"Filesystem got created from kubernetes Side")
+        k8s_validate = True
+    else:
+        log.error("Filesystem was not create at Kubernetes Side")
+        return False
+    return True if (ceph_validate and k8s_validate) else False
+
+
+def create_multiple_rbd_storageclasses(count=1, ):
+    """
+    Function for creating multiple rbd storageclass
+    :param count:
```

Comment:
fix docstring and add a description of count=1

Reply:
Done